cwms_radar_api/src/main/java/cwms/radar/formatters/json/JsonV1.java

```diff
@@ -42,7 +43,7 @@
 				   Clob.class,
 				   Clobs.class,
 				   TimeSeriesGroup.class,
-				   RecentValue.class
+				   RecentValue.class, SpecifiedLevel.class
```

Comment:
I thought JSONv1 was specifically for the calls that returned JSON generated by CWMS itself inside of Oracle, whereas v2 was for Radar-generated JSON?
I could be mistaken, as my experience with it was wrt to TimeSeries data only, which does have a CWMS generated version.

Reply:
I wasn't aware of that distinction but it sounds plausible.  Its really easy to move these from v1 to v2.  I'll fix it.

Reply:
That was the intent. slightly confusing for new things perhaps but simplifies our design... I hope.